linera-views/src/register_view.rs

```diff
@@ -77,11 +88,13 @@ where
             }
             self.stored_hash = hash;
         }
+        self.delete_storage_first = false;
         Ok(())
     }
 
     fn clear(&mut self) {
-        self.update = Some(Box::default());
+        self.delete_storage_first = true;
+        self.update = None;
```

Comment:
Shoot, I was wrong with `self.update = None` here because it breaks `get(&self)` (returning `self.stored_value` instead of `T::default()`).

Anyway this change is not needed because we are now clearing `self.delete_storage_first` in `set()` and `get_mut()`.

Probably we should add a comment
```
// Setting `self.update` to the default value is required for `self.get()`
```